src/main/resources/application.yml

```diff
@@ -30,11 +29,14 @@ spring:
 
   jpa:
     hibernate:
-      ddl-auto: update
+      ddl-auto: create
```

Comment:
This was not a case for the integration tests, the sequence of the primary keys was not created.

Reply:
OK, let's leave it in that case. From what I've read I think this leads to all previous data being dropped on application start. We still have it on 'auto' in application-production.yml though, so it shouldn't be a big issue.

Reply:
url: jdbc:h2:mem:soprafsdb;MODE=PostgreSQL;IGNORECASE=TRUE;INIT=CREATE SCHEMA IF NOT EXISTS soprafs21

As far as I can see we have in-memory DB, so the previous data will be dropped anyway. Or do I miss something?

Reply:
I'm being stupid today, I had the `mem:soprafsdb` part changed to use a file instead of in-memory at some point when I was setting things up but then changed it again b/c I had some issue before pushing it and forgot about it, sorry.

Reply:
If you change it to file-based DB, please keep it in build folder for now.